app/scripts/modules/core/src/presentation/layout/ObjectRow.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import styles from './ObjectRow.module.css';
+
+interface IObjectRowProps {
+  icon: string;
+  title: string;
+}
+
+export const ObjectRow = ({ icon, title }: IObjectRowProps) => {
+  const depth = 0;
```

Comment:
Does it make sense to have the depth passed in as an optional prop that defaults to 0? This way the parent component can control the placement of the child component. It also makes the component API more versatile for other use cases of a row that just has an icon and title.

Reply:
I don't disagree, but for now we're not worrying too much about component APIs https://github.com/spinnaker/deck/pull/8032#issuecomment-598345196

Expect these things to be up in a PR for proper review when we [decide to graduate them](https://github.com/spinnaker/deck/pull/8032#discussion_r391713997) back into `core/presentation`